metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -229,7 +229,9 @@ object SbtBuildTool {
   private def bloopPluginDetails(version: String): PluginDetails = {
     val resolver =
       if (isSnapshotVersion(version))
-        Some("""Resolver.bintrayRepo("scalacenter", "releases")""")
+        Some(
+          """"Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots""""
```

Comment:
I know the [docs](https://www.scala-sbt.org/1.x/docs/Resolvers.html#Maven+resolvers) show it this way, but do you think we should do this or do:

```
Resolver.sonatypeOssRepos("snapshot")
```

Reply:
Changed, and also for Metals snapshots since it's the same thing.

Reply:
I think the shorter option is nicer and it shouldn't matter for the users. Thanks!

Reply:
Ah I had to cancel the job. Since `sonatypeOssRepos` is actually a Vector we need to change the `+=` to a `++=`.

Reply:
My bad! I changed it and actually moved everything to one place, so that we get the full `resolvers ++= ..` in one place.

Reply:
It seems `sonatypeOssRepos` is not present in some already sbt versions :/ Should we go back to the first version then?

Reply:
Reverted!